docs/ref/migration-operations.txt

```diff
@@ -345,6 +345,12 @@ set ``atomic=False``.
     ``schema_editor.connection.alias``, where ``schema_editor`` is the second
     argument to your function).
 
+.. warning::
+
+    When migrating data across two apps, ``dependencies`` should include the target
```

Comment:
"should include the target app" seems not precise enough to me - maybe "should include the latest migration in the target app"